Godeps/Godeps.json

```diff
@@ -8,7 +8,7 @@
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
 			"Comment": "null-5",
-			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
+			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
```

Comment:
This is doing the same bizarre single quotes for me locally too. Any idea why? I assume it's not going to cause any problems going forward?